examples/Overlay.js

```diff
@@ -113,7 +113,7 @@ const OverlayExample = React.createClass({
           target={ props => findDOMNode(this.refs.target)}
         >
           <ToolTip>
-            I'm placed to the: <strong>{this.state.placement}</strong>
```

Comment:
Yeah... frankly I'm surprised the GH syntax highlighting fails so badly here. Oh well.